config/manifests/bases/awx-operator.clusterserviceversion.yaml

```diff
@@ -387,6 +387,12 @@ spec:
         - urn:alm:descriptor:com.tectonic.ui:advanced
         - urn:alm:descriptor:com.tectonic.ui:string
         - urn:alm:descriptor:com.tectonic.ui:fieldDependency:service_type:LoadBalancer
+      - displayName: LoadBalancer Class
+        path: loadbalancer_class
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:string
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:service_type:LoadBalancer
```

Comment:
It looks like this doesn't actually do anything.  I don't see it in these docs either:
* https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md

Since we don't know what options will be available on various cloud providers we probably don't want to use enum either.  

This is how it currently displays.  

![image](https://github.com/ansible/awx-operator/assets/11698892/e7b985ac-f97a-4718-9fc2-a909177fb9af)

Reply:
tl;dr: you can remove this line.  
```suggestion
```